src/pages/home/categories-list/category-item/tests/category-item.spec.js

```diff
@@ -13,12 +15,14 @@ const mockDispatch = jest.fn();
 
 useDispatch.mockReturnValue(mockDispatch);
 
-const wrapper = shallow(
-  <CategoryItem categoryName='test' categoryImageUrl='test' categoryUrl='test' />
-);
-
 describe('CategoryItem component tests', () => {
-  it('Should render CategoryItem', () => {
-    expect(wrapper).toBeDefined();
+  const { container } = render(
+    <BrowserRouter>
+      <CategoryItem categoryName='testName' categoryImageUrl='testImageUrl' categoryUrl='testUrl' />
+    </BrowserRouter>
+  );
+
+  test('Should render CategoryItem', () => {
+    expect(container).toBeInTheDocument();
```

Comment:
Can we restructure the test by doing a test on something more specific like a button, input field or text content?

Reply:
Yes, i will try something